src/chain.h

```diff
@@ -140,6 +142,10 @@ class CBlockIndex
     //! Verification status of this block. See enum BlockStatus
     unsigned int nStatus;
 
+    //! Branch ID of this block. Only accurate if block validity is at least SCRIPTS.
+    //! Persisted at each activation height, memory-only for intervening blocks.
+    uint32_t nBranchId;
```

Comment:
I will probably change this to `nBlockBranchId`, to disambiguate from `nFormatBranchId` in transactions.

Reply:
Elsewhere we decided to use "consensus branch ID" for this "type", so I'll use `nConsensusBranchId` here.